CONTRIBUTING.md

```diff
@@ -23,12 +23,10 @@ Thanks for your interest in contributing to Ionic's documentation! :tada: Check
 In order to run the documentation locally, install the dependencies and run the development server:
 
 ```sh
-$ npm install --legacy-peer-deps
-$ npm start
+npm install
+npm start
 ```
 
-> **Note**: certain versions of npm (5-8) and Node.js (10-16) are required to run certain scripts.
```

Comment:
The Ionic logo doesn't show up in the header on the "Native" page anymore: https://ionic-docs-git-docusaurus-update-ionic1.vercel.app/docs/native

Reply:
Can you check if this is still an issue? I'm seeing the logo.

Reply:
Logo looks correct now, though I see some duplicate icons: 
![image](https://github.com/ionic-team/ionic-docs/assets/2721089/c756b1a2-aab0-4421-bf24-0569f4db377a)